guides/welfare/Leave and Time Off

```diff
@@ -1,17 +1,19 @@
 Here is a list of the time off types in HiBob and some examples of what they should be booked for :
 
 ## Holiday 
-We have adopted a flexible holiday policy, where each team member is given the flexibility to take whatever holiday they need.  Please see our flexible holiday page here.
+We have adopted a flexible holiday policy, where each team member is given the flexibility to take whatever holiday they need.  
+Please see our flexible holiday page [here](https://github.com/madetech/handbook/blob/main/benefits/flexible_holiday.md)
 
 ## Sick
-We recognise that for our team to perform at the highest possible level, they need to be healthy in mind and body. Please see our sick policy page here.
+We recognise that for our team to perform at the highest possible level, they need to be healthy in mind and body. 
+Please see our sick policy page [here](https://github.com/madetech/handbook/blob/main/guides/welfare/sick_leave.md)
 
 ## Medical Leave 
 Medical leave should be used when you need some time off for a medical appointment or medical procedure, where you are not able to flex your time to make up any hours.  
 When you book this leave in HiBob, please select the reason for your medical leave so that we can make sure we put any support in place for you. Your line manager will need to approve this leave.
 
 ## Learning Time 
-This is for your personal learning time that can be used flexibly, more information can be found here.
+This is for your personal learning time that can be used flexibly, more information can be found [here](https://github.com/madetech/handbook/blob/main/guides/learning/README.md)
```

Comment:
This is for your personal learning time that can be used flexibly, more information on [learning time](https://github.com/madetech/handbook/blob/main/guides/learning/README.md)